html-syntax/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+
+ <head>
+ <title>HTML Syntax</title>
+ </head>
+
+ <body>
+ <p>
```

Comment:
very close and I am sorry for not being more clear. You don't want to wrap the other elements with a p tag, just the copyright text below on line 13.